modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -91,7 +115,8 @@ export default {
     return {
       menuItems: [],
       createMenu: [],
-      patches: []
+      patches: [],
+      editMode: window.sessionStorage.getItem('aposEditMode') === 'true'
```

Comment:
sessionStorage used to persist between pages for the lifetime of the tab per discussion with Stu and Ben.